example/tools/lint/linters.bzl

```diff
@@ -71,3 +72,12 @@ ktlint = lint_ktlint_aspect(
 )
 
 ktlint_test = lint_test(aspect = ktlint)
+
+clang_tidy = lint_clang_tidy_aspect(
+    binary = "@@//tools/lint:clang_tidy",
+    config = "@@//:.clang-tidy",
```

Comment:
curious, can a subdirectory in a monorepo override some settings for their project? If they have a nested .clang-tidy file, is the tool expected to resolve the nearest-ancestor file, and then do settings from parent directories still apply?

Reply:
A .clang-tidy file in any directory can be specified here, I am specifying this absolute path on the command-line to clang-tidy -- there is no search. It would be interesting to see if I could set it up to allow different configs per-directory.

Reply:
Yes, I've added this support as an option. The setup is rather clumsy, you need to add all the submodule .clang-tidy files to the global aspect. It would be nicer if these could be declared in each nearest BUILD file.